compiler/aarch64/codegen/OMRCodeGenerator.cpp

```diff
@@ -656,6 +656,9 @@ bool OMR::ARM64::CodeGenerator::getSupportsOpCodeForAutoSIMD(TR::ILOpCode opcode
             return true;
          else
             return false;
+
+      case TR::vdsqrt:
+         return ((dt == TR::Float) || (dt == TR::Double));
```

Comment:
I agree `return (condition);` is simple and readable.
Other lines in this function use the style like `(dt == TR::Float || dt == TR::Double)` for the condition.

Reply:
I think I will change other lines to this style in the separate issue.